sources/github/helpers.py

```diff
@@ -23,16 +23,23 @@ def _get_auth_header(access_token: str) -> StrAny:
 # Rest API helpers
 #
 def get_rest_pages(access_token: str, query: str) -> Iterator[List[StrAny]]:
-    url = REST_API_BASE_URL + query
+    def _request(page_url: str) -> requests.Response:
+        r = requests.get(page_url, headers=_get_auth_header(access_token))
+        print(
+            f"got page {page_url}, requests left: " + r.headers["x-ratelimit-remaining"]
+        )
+        return r
+
+    next_page_url = "https://api.github.com" + query
```

Comment:
use REST_API_BASE_URL instead of "https://api.github.com"

Reply:
done.